zeus/device/cpu/rapl.py

```diff
@@ -86,6 +177,10 @@ def __init__(self, path: str) -> None:
                 "Error reading package max energy range"
             ) from err
 
+        self.wraparound_tracker = RaplWraparoundTracker(
+            self.energy_uj_path, self.max_energy_range_uj
+        )
```

Comment:
QQ: Is this initialized (and thus a process is spawned) when the user first calls `get_cpus`?

Reply:
Yup, It would be like `get_cpus` -> `RAPLCPUs` -> `RAPLCPU` -> `RAPLFile` -> `RaplWraparoundTracker`